documents/motions/21.2022.1 - WCA Documents.md

```diff
@@ -1,27 +1,28 @@
 # SUBMISSION OF PROPOSED MOTION
 
-**Motion number:** 21.2022.1  
+**Motion number:** 21.2024.1  
 **Subject:** WCA Documents  
 **Intent:** Establish priorities and amendment procedures of WCA Documents  
 **Submitted by:** Motion has been removed and all responsibilities will be absorbed by the WCA Results Team.  
-**Date:** May 1, 2022  
+**Date:** TBC, 2024
 
 # Motion
 
-1. In case of any dispute between two WCA Documents the following order of priority shall be established:
+1. WCA Documents are comprised of:
    1. WCA Bylaws
    2. WCA Motions
    3. WCA Regulations and Guidelines
    4. WCA Policies
-   5. Other WCA material
-2. In case of any dispute between two WCA Documents with the same priority, the WCA Board shall determine which document takes precedence.
-3. In case of any dispute between two translations of the same document the American English version shall prevail
-4. Amendments of WCA Documents:
+   5. Other WCA material - as listed on the Documents page of the WCA Webiste.
+2. In case of any dispute between two WCA Documents the order of priority shall be established in line with the order of documents in section 1 of this motion.
+3. In case of any dispute between two WCA Documents with the same priority, the WCA Board shall determine which document takes precedence.
+4. In case of any dispute between two translations of the same document the American English version shall prevail.
```

Comment:
```suggestion
4. In case of any dispute between two translations of the same document the American English version must prevail.
```

Reply:
Perhaps "The American English version is definitive"? "Prevail" just strikes me as a weird word.